api/routes/api.php

```diff
@@ -48,6 +48,8 @@
         Route::get('/{id}', 'EventTypeController@getEventTypeById');
         Route::put('/{id}', 'EventTypeController@update');
         Route::put('/{id}/disabled', 'EventTypeController@changeDisabledById');
+        Route::post('/custom-fields', 'EventTypeController@saveCustomFieldsByEventTypeId');
+        Route::put('/custom-fields', 'EventTypeController@updateCustomFieldsByEventTypeId');
```

Comment:
You should follow REST here, pass the event type id via query instead of the request body.